modules/op-about-podtemplate.adoc

```diff
@@ -0,0 +1,53 @@
+// This module is included in the following assemblies:
+// * about/understanding-openshift-pipelines.adoc
+
+[id="about-podtemplate_{context}"]
+= Pod templates
+
+Optionally, you can define a _pod template_ in a `PipelineRun` or `TaskRun` custom resource (CR). You can use any parameters available for a `Pod` CR in the pod template. When creating pods for executing the pipeline or task, {pipelines-shortname} sets these parameters for every pod.
+
+For example, you can use a pod template to make the pod execute as a user and not as root.
+
+For a pipeline run, you can define a pod template in the `pipelineRunTemplate.podTemplate` spec, as in the following example:
+
+.Example `PipelineRun` CR with a pod template
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: mypipelinerun
+spec:
+  pipelineRef:
+    name: mypipeline
+  taskRunTemplate:
+    podTemplate:
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 1001
+----
+
+[NOTE]
+====
+In the earlier API version `v1beta1`, the pod template for a `PipelineRun` CR was specified in the `podTemplate` spec, directly in the `spec:` section. This format is not supported in the `v1` API.
```

Comment:
```suggestion
In the earlier API version `v1beta1`, the pod template for a `PipelineRun` CR was specified in the `podTemplate`, directly in the `spec:` section. This format is not supported in the `v1` API.
```
actually that spec word along with podTemplate is confusing and for me i understood as podtemplate as spec